pandas/_libs/window.pyx

```diff
@@ -1348,6 +1348,7 @@ def roll_quantile(ndarray[float64_t, cast=True] input, int64_t win,
         bint is_variable
         ndarray[int64_t] start, end
         ndarray[double_t] output
+        double vlow, vhigh
 
     if quantile < 0.0 or quantile > 1.0:
```

Comment:
Yes, but your patch does alter the valid inputs, and `quantile == 1.0` is no longer a valid one. It's more of a cosmetic thing, because the Python wrapper never calls this with `quantile=1.0`, but I think it is still worth making the change.